tools/docker/docker-compose.yml

```diff
@@ -35,5 +50,45 @@ services:
         depends_on:
             - db
 
+    char:
+        image: "rathena:local"
+        container_name: "rathena-char"
+        command: "-c ./char-server"
+        ports:
+            - "6121:6121" # char server
+        volumes:
+            - "../..:/rathena" # mount git repo directory inside container
+            - "./asset/inter_conf.txt:/rathena/conf/import/inter_conf.txt" # load db connection
+            - "./asset/char_conf.txt:/rathena/conf/import/char_conf.txt"   #localdev login-char relation
+            - "./asset/map_conf.txt:/rathena/conf/import/map_conf.txt"     #localdev char-map relation
+        init: true # helps with signal forwarding and process reaping
+        tty: true
+        stdin_open: true
+        build:
+            context: .
+            dockerfile: Dockerfile
+        depends_on:
+            - login
+
+    map:
+        image: "rathena:local"
+        container_name: "rathena-map"
+        command: "-c ./map-server"
```

Comment:
these should probably be using the absolute path just for clarity's sake 
`command: "-c /rathena/map-server"`